guides/request_lifecycle.md

```diff
@@ -265,6 +273,8 @@ For the last piece of this puzzle, we'll need a new template. Since it is for th
 
 To do that, we'll use the special EEx tags for executing Elixir expressions: `<%=  %>`. Notice that the initial tag has an equals sign like this: `<%=` . That means that any Elixir code that goes between those tags will be executed, and the resulting value will replace the tag in the HTML output. If the equals sign were missing, the code would still be executed, but the value would not appear on the page.
 
+Remember our templates are written in HEEx (HTML+EEx). HEEx is a superset of EEx which is why it shares the `<%= %>` syntax.
+
```

Comment:
New users *might* get tripped up by "EEx tags" vs "aren't we writing HEEx?" so this tries to head that off at the pass.